setup.py

```diff
@@ -70,7 +70,7 @@
     },
     # Packaging
     package_dir={"": "src"},
-    packages=find_packages("src") + namespace_packages,
+    packages=find_namespace_packages(where="src") + namespace_packages,
```

Comment:
Do we still need this `namespace_packages` with this new implementation?

Reply:
it seems we don't. I removed it.